src/test/java/com/conveyal/datatools/manager/jobs/FlexMergeFeedsJobTest.java

```diff
@@ -169,4 +134,11 @@ private FeedVersion regionallyMergeVersions(Set<FeedVersion> versions) {
         LOG.info("Regional merged file: {}", mergeFeedsJob.mergedVersion.retrieveGtfsFile().getAbsolutePath());
         return mergeFeedsJob.mergedVersion;
     }
+
+    /**
+     * Helper method to confirm that the sum of the two feed table rows match the merged feed table rows.
+     */
+    private void assertRowCount(TableLoadResult feedOne, TableLoadResult feedTwo, TableLoadResult feedMerged, String message) {
+        assertEquals(feedOne.rowCount + feedTwo.rowCount, feedMerged.rowCount, message);
```

Comment:
Nice! Could you move the text "<entity> count for merged feed should equal the <entity> sum for the versions merged" inside this function, and only pass a short string `entity`?

Reply:
Yep, updated.